app/src/hooks/useParaspellApi.ts

```diff
@@ -400,3 +408,41 @@ const useParaspellApi = () => {
 }
 
 export default useParaspellApi
+
+interface SendMetricsParams {
+  params: TransferParams
+  txId: string
```

Comment:
Let's better do this
```suggestion
  txId?: string
```

And then you don't need to do `... ?? ''`